Cargo.toml

```diff
@@ -18,6 +18,10 @@ rand = "0.7"
 serde = { version = "1", features = ["derive"] }
 serde_json = "1"
 sha2 = "0.9"
+shiplift =  {git = "https://github.com/softprops/shiplift" } 
```

Comment:
Let's hope they are cutting a release soon! We can't release with `git` dependencies so if we merge this before hand, we will need to do some manual work to exclude this code on every release. That shouldn't be too hard to so it is not actually a blocker for merging this PR! 

GitFlow for the win! :grin:

Reply:
To make this easier, could you squash all of the commits into a single one so I can just revert that when we make releases?